scheduler/test/cook/test/rest/api.clj

```diff
@@ -2315,3 +2315,13 @@
   (is (api/valid-date-str? "20180101"))
   (is (not (api/valid-date-str? "2018-01-01")))
   (is (not (api/valid-date-str? "20180132"))))
+
+(deftest test-match-default-containers
+  (let [default-containers
+        [{:pool-regex "^foo$" :container {:foo 1}}
+         {:pool-regex ".*" :container {:bar 2}}
+         {:pool-regex "^baz$" :container {:baz 3}}]]
+    (is (api/get-default-container-for-pool default-containers "foo") {:foo 1})
+    (is (api/get-default-container-for-pool default-containers "bar") {:bar 2})
+    (is (api/get-default-container-for-pool default-containers "baz") {:bar 2}))
```

Comment:
I think these are all missing a call to `=`

Reply:
I agree. Good catch.

Reply:
@scrosby I still don't see the calls to `=`

Reply:
I did it. Not sure where the commit got lost at. In any case. I've validated its done.